src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/AvailabilityDataType.cs

```diff
@@ -20,7 +21,7 @@ public AvailabilityDataType()
 
             Field<BooleanGraphType>("IsTrackInventory", resolve: context => context.Source.IsTrackInventory);
 
-            Field<ListGraphType<InventoryInfoType>>("inventories", resolve: context => context.Source.InventoryAll);
+            Field(GraphTypeExtenstionHelper.GetComplexType<ListGraphType<InventoryInfoType>>(), "inventories", resolve: context => context.Source.InventoryAll);
```

Comment:
Can we create an `ExtendedField` method extension?

Reply:
``` csharp
ExtendedField<ListGraphType<InventoryInfoType>>("inventories", resolve: context => context.Source.InventoryAll);
```

Reply:
What do you think about it?

Reply:
Yeah!  This is a great idea! This would significantly simplify the declaration syntax.